tools/abel/README.md

```diff
@@ -1,191 +1,219 @@
+ * Run Instance Validator (with optional telemetry validation) on the building configuration file during the conversion process to ensure there are no errors in the file prior to attempting onboarding.
+* Convert a building configuration file to a Google Sheet to make edits on.
+
 
-* [Setup and Installation](#setup-and-installation)
+## User Guide Contents
+
+* [Getting Started](#getting-started)
+  *	[Set Up a Tooling Environment](#set-up-a-tooling-environment)
+  *	[Obtain a GCP OAuth Client Credential](#obtain-a-gcp-oauth-client-credential)
 * [Using ABEL](#using-abel)
-  * [Update Workflow](#update-workflow)
-  * [Initialization Workflow](#initialization-workflow)
-  * [Split Workflow](#split-workflow)
+  *	[General Use](#general-use)
+    *	[Update your Repository Clone and Activate your Virtual Environment](#update-your-repository-clone-and-activate-your-virtual-environment)
+    *	[ABEL Arguments](#abel-arguments)
+  * [Updating an Existing Building](#updating-an-existing-building)
+    *	[Create a Spreadsheet and Convert it to a Building Configuration File](#create-a-spreadsheet-and-convert-it-to-a-building-configuration-file)
+    *	[Generate a Building Configuration File from an Existing Spreadsheet](#generate-a-building-configuration-file-from-an-existing-spreadsheet)
+  * [Initializing a New Building](#initializing-a-new-building)
+
+
+## Getting Started
+
+ABEL is a Python tool that must be run in the command line. Before starting the setup and installation process outlined below, please ensure that the following dependencies are met:
+
+* You are running Python 3.9 or higher
+* You have a local clone of the Digital Buildings repository downloaded on your computer
+  * A local clone can be created by running the following in your CLI:
+    
+    ```
+    git clone https://github.com/google/digitalbuildings.git
+    ```
 
-## Setup and Installation
+The following steps only need to be done on initial setup. Any steps that must be repeated with use will be outlined in the [Using ABEL](#using-abel) section.
 
-Before starting the setup and installation process, please ensure that the dependencies are met:
 
-1. You are running **Python 3.9** or higher
+### Set Up a Tooling Environment
 
-2. you have installed the [Google Cloud CLI](https://cloud.google.com/sdk/docs/install)
+A virtual tooling environment ([venv](https://docs.python.org/3/library/venv.html)) must be made to create a separate, isolated instance of the Python runtime for a project, with its own packages. To create this environment, follow the following steps:
 
-The total setup process should only take about 15 minutes. The process is broken into two parts:
+1. In the command line, navigate to the Digital Buildings tooling library cloned on your computer. If the Digital Buildings repository is not in your home directory, replace “~” with the folder it exists within.
+```
+cd ~/digitalbuildings/tools/
```

Comment:
This is a Windows OS specific command, please also include MacOS / Linux

Reply:
What's the difference for linux/mac? This is the command I've been using in my remote desktop and I'm not seeing much different online.